spec/Functions.tex

```diff
@@ -767,6 +764,43 @@ \subsection{The Ref Return Intent}
 \end{chapeloutput}
 \end{chapelexample}
 
+
+\subsection{The Const Ref Return Intent}
+\label{Const_Ref_Return_Intent}
+\index{functions!const ref keyword and@\chpl{const ref} keyword and}
+\index{const ref (return intent)@\chpl{const ref} (return intent)}
+\index{intents!const ref return@\chpl{const ref} return}
+
+The \chpl{const ref} return intent is also available. It is a restricted
+form of the \chpl{ref} return intent. In particular, it uses a similar
+mechanism but the function call cannot be assigned to. The \chpl{const ref}
+return intent should be viewed as a performance optimization. It
+indicates that the returned value does not need to be copied because the
+returned value will not be destroyed when the function returns.
+
```

Comment:
Just to clarify:  In some contexts returning with ref intent could support performance optimizations that avoid
copies.  Returning with const ref rather than ref merely clarifies the object being referenced cannot be modified
by the caller.